### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by karanja-joseph**
- Authentication and Authorization done
- Admin menu and patient menu done
- layout for content and notification 2 b done... 
- 
**Issues Tackled:**

- [Backend | Patients sign up and login the system](https://github.com/karanja-joseph/psychiatrist-booking-app/issues/8)
- [Frontend | Patients sign up and login the system](https://github.com/karanja-joseph/psychiatrist-booking-app/issues/7)
- [Backend | Authentication and authorization controls](https://github.com/karanja-joseph/psychiatrist-booking-app/issues/13)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "description": "_17th September 2022_",
   "main": "server.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "npx nodemon server.js"
```
Comment (mwchben) [inline package.json:8]: @karanja-joseph nice thought instead of using `nodemon` every time.

File: models/patientModel.js [hunk 2]
```diff
@@ -14,10 +14,26 @@ const patientsSchema = new mongoose.Schema(
             type: String,
             required: true,
         },
-    },
-    {
-        timestamps: true,
-    }
+        isDoctor: {
```
Comment (mwchben) [inline models/patientModel.js:17]: we need psychiatrist as a different entity

**Review by mwchben — COMMENTED** [review]
find comments

File: package.json (hunk 1 above)
Comment (karanja-joseph) [reply package.json:8]: Thanks!

File: models/patientModel.js (hunk 2 above)
Comment (karanja-joseph) [reply models/patientModel.js:17]: registering different like the patients?

Comment (mwchben) [reply models/patientModel.js:17]: Yes, that way.. we can have an admin.

**Pull request merged**